teams/risc-v.toml

```
@@ -0,0 +1,10 @@
+name = "riscv"
```

```suggestion
name = "risc-v"
```

The name here should be the same as the file name :)